### ISSUE TITLE
Missing hashcat-utils

### ISSUE THREAD

**Issue by agenta007** [staz0t/hashcatch#22]
I am trying to setup hashcatch to run on my kali linux, followed the steps as instructed. Hashcatch recognizes my network adapter and successfully puts it into monitor mode. At the next step setup quits, because its missing hashcat-utils. I've been searching for a fix, but can't figure out how to make it to install.

What I've tried
changing the $PATH variable - i added both /usr/lib/hashcat-utils/ and usr/share/hashcat-utils folders, which later i found out are the same. Tried to change $path only on current shell and then also tried to change the .bashrc file. I also tested if the paths are properly added with echo $PATH, both folders always appear even after reboot.
I have all other dependecies which are aircrack-ng, hcxtools and jq, which i check with command -v <command_name>
I've also tried putting the ~/lib/hashcat-utils/ folder into /usr/bin/
I can't figure it out please help me

**Comment by staz0t** [comment]
Hi,
Hashcatch uses the cap2hccapx command from hashcat-utils. So, you'll get that error only if this command isn't accessible via path.

Try ```command -v cap2hccapx``` and see if it's accessible.

**Comment by agenta007** [comment]
Exactly what I am doing "command -v cap2hccapx" has no output, but it should print something, folder with hashcat-util is in my $PATH

**Comment by staz0t** [comment]
Is there a cap2hccapx file in your hashcat-util directory? And are the other files in that directory accessible via path?

**Comment by Cilo13** [comment]
Hi, I'm having the same issues, i've setup my PATH correctly and can run cap2hccapx from any folder. 

kali@artemis:~$ cap2hccapx
usage: cap2hccapx input.pcap output.hccapx [filter by essid] [additional network essid:bssid]

Still keep getting the error 

[*] Starting hashcatch setup
[!] WARNING! Continuing with setup will rewrite your existing config file, db file, and your handshakes directory! Take a backup if necessary
[!] Do you want to proceed? [y/N]: Y
[*] Enter your wireless interface: wlan1
[*] Trying to set the given interface to monitor mode
[+] The adapter is working in monitor mode!
[!] The following packages are missing. Please ensure that you have installed them properly before starting hashcatch
        hashcat-utils

**Comment by staz0t** [comment]
If there's an issue with PATH or missing packages, it's definitely not a problem with hashcatch. Just try checking if you can access the program from the terminal first. If you can, hashcatch can. You will not face the error. If you cannot access the program from the terminal without using the full path, then it's likely that there's some issue with the configuration of PATH, or if the program exists. Like I said the only way to fix it is by checking if you can run it. `command -v cap2hccapx` or `which cap2hccapx` will help.

**Comment by gazanfarus** [comment]
For those, who still have problems - it was because of the file name check (it checks for cap2hccapx instead of cap2hccapx.bin) Just remove file extension to fix the problem. I just copied the modified file into usr/bin directory

**Comment by staz0t** [comment]
Damn, that explains! Thanks, @gazanfarus!

**Comment by Lvc4** [comment]
i have the same issue. whe  trying  `command -v cap2hccapx` no output and i have renamed the file from cap2hccapx.bin to cap2hccapx but the same error.
`[!] The following packages are missing. Please ensure that you have installed them properly before starting hashcatch
        hashcat-utils`

**Comment by D0TWav** [comment]
```
Make sure that cap2hcapx.bin or cap2hcapx exists.
Even after ensuring that cap2hcapx could be ran in the terminal i still ran across the errors, after manually installing wlanhcxinfo from https://gitlab.com/kalilinux/packages/hcxtools I was able to run it. It appears that the updated hcxtools no longer includes the wlanhcxinfo tool, as it was depreciated.

https://raw.githubusercontent.com/ZerBea/hcxtools/master/changelog
10.09.2020
==========
removed deprecated wlanhcx2ssid, wlanhcxinfo aand wlanhcxcat

I also REMOVED DEPENDENCY CHECKING because I ensured that cap2hcapx and wlanhcxinfo existed inside my $PATH.

To summarize/TLDR: 

> sudo apt-get install hashcat-utils
>PATH=$PATH:/usr/share/hashcat-utils
>git clone https://gitlab.com/kalilinux/packages/hcxtools.git
>cd hcxtools && make && sudo make install
>command -v wlanhcxinfo && command -v cap2hccapx 
SHOULD RETURN
/usr/local/bin/wlanhcxinfo
/usr/share/hashcat-utils/cap2hccapx
Then remove all Dependency checks for Debian and/or arch and it should work
